swagger.yaml

```diff
@@ -0,0 +1,100 @@
+openapi: 3.0.1
```

Comment:
THe OAS spec is already there: https://github.com/NeuraLegion/go-todoapp-demo/blob/main/docs/openapi3.yaml